code/__DEFINES/dynamic.dm

```diff
@@ -7,6 +7,9 @@
 /// This ruleset can only be picked once. Anything that does not have a scaling_cost MUST have this.
 #define LONE_RULESET (1 << 2)
 
+/// This ruleset has lazy loaded templates that must be loaded if the ruleset is selected
+#define RULESET_USES_LAZY_TEMPLATES (1 << 3)
```

Comment:
This flag doesn't feel necessary

Reply:
I can have it just check for the list existing?

Reply:
You don't even haev to do that, looping over null does nothing